### ISSUE TITLE
Non-book instructions and examples in DCRMR

### ISSUE THREAD

**Issue by deborahjleslie** [rbms-bsc/DCRMR#133]
In a number of places, examples are given for material types other than books. In at least one place, there is an instruction that explicitly applies to graphics. See [0.4.94.2 Still images](https://rbms-bsc.github.io/DCRMR/general-rules/Transcription/#0494-order-and-transposition); and 0.4.94.6 for an instruction brought in from DCRMC with a map example. 

I find this problematic. Since we are explicit that at this time, DCRMR only contains instructions for books, I think it will be confusing to users as to the scope of DCRMR.

I get that some of these non-book rules and examples slipped in because editors wanted placeholders for when we incorporate rules for other material types. At least, may I propose that only book examples be used?

**Comment by rarebkcat** [comment]
Re: 0.4.94.6 -- that instruction came up when Michelle, Iris, and I were working on Transcription. We concluded that the instruction could apply to modern poetry broadsides or artist's books, which straddle the line between graphic and textual resources but are cataloged as books. 

It's in GitHub at Issue #108 (referencing its old citation numbering, 0.5), which talks about the question over retaining the instruction and the decision to do so at the Editorial Group meeting of Sept. 28.

(Teh reason I remember it's at #108 is that I wrote the issue and the surrounding Transcription emails. The [email to the listserv on Sept. 27](https://mailman.yale.edu/mailman/private/rbms-ps/2020-September/000296.html) has yet more context. XD So many conversations, we have had!)
